flytekit/clis/flyte_cli/main.py

```diff
@@ -1625,11 +1625,11 @@ def _extract_files(
     # .pb
 
     results = []
-    filename_iterator = iter(file_paths)
-    for identifier_file in filename_iterator:
-        object_file = next(filename_iterator)
-        # Serialized proto files are of the form: 12_foo.bar.<resource_type>.pb
-        id, entity = _extract_pair(identifier_file, object_file, project, domain, version, patches or {})
+    for proto_file in file_paths:
+        # Serialized proto files are of the form: 12_foo.bar_1.pb
+        # Where 12 indicates it is the 12 file to process in order and 1 that is of resource type 1, or TASK.
```

Comment:
add what 2 and 3 are?  since we don't delete proto fields they shouldn't change, even if we add more in the future.

Reply:
it's defined above https://github.com/lyft/flytekit/pull/351/files#diff-43826f51b783bcfa61d9f2cba92c1f864167b02b39426746ee1dcf7f8c744436R1600